packages/create-next-app/create-app.ts

```diff
@@ -236,34 +233,69 @@ export async function createApp({
     if (typescript) {
       devDependencies.push('typescript', '@types/react', '@types/node')
     }
-    /**
-     * Install package.json dependencies if they exist.
-     */
-    if (dependencies.length) {
-      console.log()
-      console.log('Installing dependencies:')
-      for (const dependency of dependencies) {
-        console.log(`- ${chalk.cyan(dependency)}`)
+
+    if (skipInstall) {
+      console.log('Skipping dependency installation step.')
```

Comment:
If `skipInstall` is `true` then you've already logged this message earlier in Line 195. Should

Reply:
It's actually in an else block meaning it'd only be printed once. Chiming in since i was working on this :)

Reply:
like @darshkpatel said, this log is called when the `--example` flag is not passed, and the previous one when it is passed.

Reply:
That is right, thanks for the correction @darshkpatel @Pika-Pool